Azure/sonic-buildimage
# [Mellanox] [202106] remove 2x40G and 4x40G breakout modes due to no hardware support

#### Why I did it
Manual cherry-pick of https://github.com/Azure/sonic-buildimage/pull/8280
Mellanox platforms do not support 2x40G or 4x40G breakout modes.

#### How I did it
Removed these breakout modes from platform.json and hwsku.json files

#### How to verify it
Run show interface breakout and verify no 2x40G or 4x40G modes are shown in the supported breakout modes.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
